pkg/config/user_config.go

```diff
@@ -822,6 +823,9 @@ func GetDefaultConfig() *UserConfig {
 				OpenLogMenu:                    "<c-l>",
 				OpenInBrowser:                  "o",
 				ViewBisectOptions:              "b",
+				// Using shift+I because we may want to use lowercase 'i' as an
+				// alias for editing a commit.
+				QuickStartInteractiveRebase: "I",
```

Comment:
Not sure I understand the reasoning for shift-I; I'd have expected 'i', like the PR description says. Not sure why 'e' needs another alias.

Reply:
I started with 'i' but it struck me that the most intuitive thing that 'i' would do would be to start an interactive rebase from the selected commit, whereas quick-start would be more of a special case. So I wanted to give us the freedom to change that in the future without worrying about backwards compatibility.

Reply:
> the most intuitive thing that 'i' would do would be to start an interactive rebase from the selected commit,

Not sure about that; you could also argue that the most intuitive thing that 'i' would do is to do an interactive rebase of your current branch.

Thinking about this more, I'm also against calling this "quick start", as if it's just a special alternative for the "normal" way to start an interactive rebase. This **is** the normal way to start one (or it could become it), so I would rather call it "Start interactive rebase of current branch", or maybe even just "Start interactive rebase" (the tooltip could explain more). I have watched lazygit newcomers struggle to find out how to interactively rebase a branch, being surprised that they have to select a commit to rebase from.

And binding it to shift-I contributes to conveying that it's somehow special, or an expert or niche thing or something. I really think it deserves to be bound to 'i'.

Reply:
I'm happy to do that and see how users like it. I have a premonition that it _is_ a special action for the following reasons:
* this feature isn't built into the git CLI so most people coming from the CLI will expect to be specifying a base commit by default
* the rules we've defined for finding the base commit are specific to our own workflows and may not reflect the typical workflow (e.g. maybe some people expect merges in their rebases, or some people directly commit to the main branch in personal projects, or do want stacked branches to be taken into consideration etc)

Nonetheless, fortune favours the brave, so let's go with 'i', call it 'Start interactive rebase' and explain the logic in a tooltip.

Reply:
> * this feature isn't built into the git CLI so most people coming from the CLI will expect to be specifying a base commit by default

I see a lot of people do `git rebase -i master` on the command line. Which isn't the same, but comes close, at least in the intention to mean "rebase my current branch interactively".

> Nonetheless, fortune favours the brave, so let's go with 'i', call it 'Start interactive rebase' and explain the logic in a tooltip.

🎉